x-pack/plugins/cases/server/client/user_actions/connectors.ts

```diff
@@ -106,23 +107,36 @@ const getConnectorsInfo = async ({
   latestUserAction,
   actionsClient,
   userActionService,
+  logger,
 }: {
   caseId: string;
   connectors: CaseConnectorActivity[];
   latestUserAction?: SavedObject<CaseUserActionResponse>;
   actionsClient: PublicMethodsOf<ActionsClient>;
   userActionService: CaseUserActionService;
+  logger: CasesClientArgs['logger'];
 }): Promise<GetCaseConnectorsResponse> => {
   const connectorIds = connectors.map((connector) => connector.connectorId);
-
-  const [pushInfo, actionConnectors] = await Promise.all([
-    getPushInfo({ caseId, activity: connectors, userActionService }),
-    actionsClient.getBulk(connectorIds),
-  ]);
+  const pushInfo = await getPushInfo({ caseId, activity: connectors, userActionService });
+  const actionConnectors = await getActionConnectors(actionsClient, logger, connectorIds);
 
   return createConnectorInfoResult({ actionConnectors, connectors, pushInfo, latestUserAction });
 };
 
+const getActionConnectors = async (
+  actionsClient: PublicMethodsOf<ActionsClient>,
+  logger: CasesClientArgs['logger'],
+  ids: string[]
+): Promise<ActionResult[]> => {
+  try {
+    return await actionsClient.getBulk(ids);
+  } catch (error) {
+    // silent error and log it
```

Comment:
@jonathan-buttner I had to silence the errors because if the connector is deleted but attached to a case or if you do not have permission to see the connectors the endpoint will return an error and blocks the UI. I believe is better to silence the error and fallback to the information we have for the connectors from the user actions.

Reply:
Ah ok, yeah makes sense